DockerDevSetupGuideForMacOS.md

```diff
@@ -0,0 +1,340 @@
+
+```bash
+    docker-sync-stack start # This is the equivalent of running docker-sync start followed by docker-compose up
+```
+
+### Stop docker-sync application and remove all containers/volumes
+```bash
+    docker-sync-stack clean # This is the equivalent of running docker-sync clean followed by docker-compose down
+    docker volume prune
+```
+
+### Rebuilding Images and Containers
+```bash
+    docker-compose -f docker-compose-dev.yml up --build --force-recreate  [<service_name1> <service_name2> ...]
+```
+
+or
+
+```bash
+    docker-compose -f docker-compose-dev.yml build --no-cache --pull [<service_name1> <service_name2> ...] 
+    docker-compose -f docker-compose-dev.yml up --force-recreate  [<service_name1> <service_name2> ...]
+```
+
+```bash
+
+    # Options:
+    #   --force-recreate                        Recreate containers even if their configuration and image haven't changed.
+    #   --build                                 Build images before starting containers.
+    #   --pull                                  Pull published images before building images.
+    #   --no-cache                              Do not use cache when building the image.
+    #   [<service_name1> <service_name2> ...]   Services to recreate, not specifying any service will rebuild and recreate all services
+```
+
+After rebuilding imaages and containers, start docker-sync normally  
+
+```bash
+    ctrl + c # Stop running "docker-compose up" command (containers running without sync functionality)
+    docker-sync-stack start # If this command fails to run, running a second time usually fixes the issue
+```
+
+### Useful docker-sync commands
+Reference: https://docker-sync.readthedocs.io/en/latest/getting-started/commands.html
+
+## Verify DRLS is working
+
+### Register the test-ehr
+
+1. Go to http://localhost:3005/register.
+    - Client Id: **app-login**
+    - Fhir Server (iss): **http://localhost:8080/test-ehr/r4**
+2. Click **Submit**
+
+### The fun part: Generate a test request
+
+1. Go to http://localhost:3000/ehr-server/reqgen.
+2. Click **Patient Select** button in upper left.
+3. Find **William Oster** in the list of patients and click the dropdown menu next to his name.
+4. Select **E0470** in the dropdown menu.
+5. Click anywhere in the row for William Oster.
+6. Click **Submit** at the bottom of the page.
+7. After several seconds you should receive a response in the form of two **CDS cards**:
+    - **Respiratory Assist Device**
+    - **Positive Airway Pressure Device**
+8. Select **Order Form** on one of those CDS cards.
+9. If you are asked for login credentials, use **alice** for username and **alice** for password.
+10. A webpage should open in a new tab, and after a few seconds, a questionnaire should appear.
+11. Fill out questionnaire and hit next
+12. Submit PAS Request to http://localhost:9015/fhir
```

Comment:
I can't get get dtr to authenticate with pas locally or as part of this docker set up or the production docker set ups. There's no issues with the logica servers or with the PAS AEGIS tests so I don't think this is a bug but rather something that I must missing. The issue is not a networking issue either as dtr can hit the pas metadata endpoint with no issues. If anyone reviewing can get dtr and pas to authenticate, let me know by responding to this comment.

Reply:
http vs https? Only difference I can think of between running locally and running on logicahealth